components/mentions-legales/texte-mentions-legales.jsx

```diff
@@ -21,22 +21,20 @@ function MyComponent({ classes }) {
       <hr />
 
       <p>
-        Incubateur de services numériques (beta.gouv.fr)
-        <br />
-        de la Direction interministérielle du numérique (DINUM).
+        Equipe Leximpact de L'Assemblée nationale
       </p>
-      <p>20, avenue de Ségur - 75007 Paris.</p>
+      <p>126, rue de l'Université - 75007 Paris.</p>
```

Comment:
> Seul commentaire de la PR d'origine : 

Pas certaine que ceci fasse une différence dans le réel mais sur le [site Assemblée](http://www.assemblee-nationale.fr/faq.asp), le code postal est légèrement différent :
```suggestion
      <p>126, rue de l'Université - 75355 Paris 07 SP.</p>
```